common/src/main/webapp/less/style.less

```diff
@@ -1129,6 +1129,15 @@ tfoot {
   margin-left: 0.5em;
 }
 
+.input-replacement-icon {
+  height: fit-content;
+  padding: 0.25em 0;
```

Comment:
I couldn't find the variable that is setting the padding for the inputs. I found variables with what seemed to be the right names, but the wrong values.